### PR TITLE
Use aliases for components, directives, mixins and utils

### PR DESCRIPTION
**Pull request by spiess-demos**
To not always have to write cumbersome path declarations like '../../component/component', new aliases for the components, directives, mixins and utils folders are introduced.

Also, this makes reorganizing components within folders a breeze.

Aliases start with `~~/` to differentiate the aliased modules from modules published by an organization (like '@demos-europe').

### CODE REVIEW

File: src/components/DpCheckbox/DpCheckbox.stories.mdx [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import { Canvas, Meta, Story } from '@storybook/addon-docs'
 import DpCheckbox from './DpCheckbox'
 
-<Meta
```
Comment (hwiem) [inline src/components/DpCheckbox/DpCheckbox.stories.mdx:4]: Is this not needed?

**Review by hwiem — APPROVED** [review]
thanks, very nice!
just 1 question

**Review by salisdemos — APPROVED** [review]
looks good. Thanks for the improvement. Just the Question why You decided to use `~~` instead of `@` like in demosplan-core?

**Comment by spiess-demos** [comment]
> Just the Question why You decided to use `~~` instead of `@` like in demosplan-core?

@hwiem and me discussed this yesterday. at first, we went with `@`, too. but as this character is also used as a prefix for [npm organizations](https://docs.npmjs.com/about-organization-scopes-and-packages), we found it less confusing to introduce a distinct character. As [resolve.alias takes precedence over other module resolutions](https://webpack.js.org/configuration/resolve/#resolvealias), it would not introduce any errors in using the `@`, but the visual distinction adds to readability, i think.

**Comment by spiess-demos** [comment]
Superseeded by https://github.com/demos-europe/demosplan-ui/pull/324

**Pull request closed without merging**